packages/api/src/external/aws/app-config.ts

```diff
@@ -1,26 +1,15 @@
 import {
   FeatureFlagDatastore,
+  getFeatureFlags,
   getFeatureFlagValue,
-} from "@metriport/core/dist/external/aws/app-config";
-import { MetriportError } from "@metriport/core/util/error/metriport-error";
+} from "@metriport/core/external/aws/app-config";
 import { capture } from "@metriport/core/util/notifications";
+import { MetriportError } from "@metriport/core/util/error/metriport-error";
 import { errorToString } from "@metriport/shared/common/error";
 import { Config } from "../../shared/config";
-import { Util } from "../../shared/util";
-
-const log = Util.log(`App Config - FF`);
-
-const listOfFeatureFlags: Array<keyof FeatureFlagDatastore> = [
-  "cxsWithEnhancedCoverageFeatureFlag",
-  "cxsWithCQDirectFeatureFlag",
-  "cxsWithCWFeatureFlag",
-  "cxsWithADHDMRFeatureFlag",
-  "cxsWithIncreasedSandboxLimitFeatureFlag",
-  "cxsWithNoWebhookPongFeatureFlag",
-  "commonwellFeatureFlag",
-  "carequalityFeatureFlag",
-  "oidsWithIHEGatewayV2Enabled",
-];
```

Comment:
No need to duplicate this since we're using a Zod schema to validate the FFs now. This was risky as we could forget to update this list, which would result in not validating the FFs at start-up time, potentially leading to runtime errorrs.